### PR TITLE
Spiral Order Traversal in Binary tree

### LINKED ISSUE #814: Spiral Order Traversal in Binary tree in Java and Python
Referencing #174

### PR DESCRIPTION
**Pull request by singh-shreya6**
Fixes #814 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Sample Input / Output.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:
-  Added Spiral Order Traversal in Binary Tree in other languages

#### Languages Used:
-  Java
- Python

#### Files Added:
- Tree_Spiralorder_Traversal.java
- Tree_Spiralorder_Traversal.py

### CODE REVIEW

**Comment by jainaman224** [comment]
@singh-shreya6 Please provide your input on PR template.

File: Tree_Spiralorder_Traversal/Tree_Spiralorder_Traversal.java [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+        left = null;
+        right = null;
+    }
+}
+
+class Tree_Spiralorder_Traversal
+{
+    // Function to print Binary Tree in Spiral Order
+     public static void SpiralOrderTraversal(Node root) 
+    {
```
Comment (jainaman224) [inline Tree_Spiralorder_Traversal/Tree_Spiralorder_Traversal.java]: Indentation error.

**Comment by jainaman224** [comment]
Please have a look https://www.oracle.com/technetwork/java/javase/documentation/codeconventions-141388.html.

File: Tree_Spiralorder_Traversal/Tree_Spiralorder_Traversal.java (hunk 1 above)
Comment (singh-shreya6) [reply Tree_Spiralorder_Traversal/Tree_Spiralorder_Traversal.java]: fixed

**Comment by singh-shreya6** [comment]
> @singh-shreya6 Please provide your input on PR template.

The PR template is a welcome change. It makes the PR much neater, uniform and makes it easier to comprehend the changes for both participants and mentors.

**Pull request merged**